src/package_validation_tool/matching/autotools.py

```diff
@@ -572,14 +572,40 @@ def _generate_autotools_files(self, tool_paths: Dict[str, Optional[str]]) -> boo
         """
         log.info("Generating Autotools files in source repository...")
 
-        # prepare PATH environment variable
+        # prepare PATH and PERL5LIB environment variables
```

Comment:
I don't understand why this is needed (even after reading the commit message). Could you briefly explain in the comment here? Maybe add a link to some reference.

Reply:
Ok, I got it. Autotools come with Perl modules that they need. And the standard directories where Perl looks for modules are useless to us, since we install our own Autotools and install them in a local directory. So Autotools must use the locally-installed Perl modules that are under `<autotool-bin-directory>/../share/`